Cloud release notes for May 2024 (#6223)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2024-05-releasenotes.md

# May 2024

## Key Takeaways

> * **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)

Reviewer: ```suggestion
* **Upload Custom Database** - You can now upload your Umbraco database to Umbraco Cloud. Add additional text, restoring content is right around the corner.
```

* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.

## Upload Custom Database

> Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 

Reviewer: ```suggestion
With the introduction of the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling back up and restore capabilities for Umbraco Cloud websites.. 
```

Author: THe sentence will be too long with your suggestion 🙈

I have tried to add another suggestion.

> Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 

Author: ```suggestion
By introducing the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backups and restore capabilities of Umbraco Cloud websites. 
```

> With the introduction of the Point-In-Time Recovery (PITR) feature on the new "Backups" page, we started a process of being able to backup and restore a whole Umbraco Cloud website. 

Author: ```suggestion
Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 
```

> With the introduction of the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling back up and restore capabilities for Umbraco Cloud websites.. 

Author: ```suggestion
By introducing the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backups and restore capabilities of Umbraco Cloud websites. 
```

> This release has enabled you to upload your custom Umbraco database to Umbraco Cloud. Utilizing the current backup functionality or bringing your own database is possible. We expect the possibility of restoring a database to your environment will be available in May or June 2024

Reviewer: ```suggestion
This release allows you to upload your custom Umbraco database to Umbraco Cloud. You can either utilize the current backup functionality or bring your own database. We expect the possibility to restore a database to your environment to be available in May or June 2024.
```
